contracts/modules/Module.sol

```diff
@@ -81,12 +85,25 @@ contract Module is IModule, ModuleStorage, Pausable {
         require(token.transfer(msg.sender, balance), "Transfer failed");
     }
 
-   /**
-    * @notice Reclaims ETH
-    * @dev We duplicate here due to the overriden owner & onlyOwner
-    */
+    /**
+     * @notice Reclaims ETH
+     * @dev We duplicate here due to the overriden owner & onlyOwner
+     */
     function reclaimETH() external {
         _onlySecurityTokenOwner();
         msg.sender.transfer(address(this).balance);
     }
+
+    /**
+     * @notice Deduct the usage cost
+     */
+    function _deductUsageFee() internal {
+        uint256 _usageCost = IModuleFactory(factory).usageCostInPoly();
+        if (_usageCost > 0) {
+            address factoryOwner = IModuleFactory(factory).owner();
+            require(factoryOwner != address(0), "Invalid owner");
```

Comment:
I guess this means we can break modules (that have usage fees) by setting the factoryOwner to 0x0? I wonder whether we should just add this constraint to `if (_usageCost > 0)` instead so that if someone sets the factory owner to 0x0 they lose fees, but don't break the module?

Reply:
That makes sense